source/Mission.cpp

```diff
@@ -354,7 +354,8 @@ void Mission::Save(DataWriter &out, const string &tag) const
 			out.Write("stopover", planet->Name(), "visited");
 		
 		for(const NPC &npc : npcs)
-			npc.Save(out);
+			if(npc.PassedSpawn() && !npc.PassedDespawn())
```

Comment:
This again suggests to me that the check on passing spawn only gets done once. Is that really the intention?

Reply:
Yes. `to spawn` is only a matter of the NPCs being created once the mission is accepted. It is not a matter of "the mission is accepted, so now if at any point when the mission is active these conditions are met, the NPCs will spawn."